### PR TITLE
enh(front): update style to css modules

### PR DESCRIPTION
**Pull request by kduret**
<h2> Type of change </h2>

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [x] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

<h2> Target serie </h2>

- [ ] 2.8.x
- [ ] 18.10.x
- [x] 19.04.x (master)

<h2> How this pull request can be tested ? </h2>

Navigate in centreon web and check if the style is not broken

### CODE REVIEW

File: www/front_src/src/components/form-fields/CheckboxField.js [hunk 1]
```diff
@@ -32,18 +34,17 @@ const CheckboxField = ({
...
         {label}
         {info}
       </label>
     </div>
     {error ? (
-      <div class="invalid-feedback">
-        <i class="fas fa-exclamation-triangle" />
-        <div class="field__msg  field__msg--error">{error}</div>{" "}
+      <div className={styles["invalid-feedback"]}>
+        <div className={classnames(styles["field__msg"], styles["field__msg--error"])}>{error}</div>{" "}
```
Comment (sc979) [inline www/front_src/src/components/form-fields/CheckboxField.js]: is the "exclamation-triangle" picture useless ?

Comment (sc979) [reply www/front_src/src/components/form-fields/CheckboxField.js]: or is it already displayed using the "styles["field__msg--error"]" ?

File: www/front_src/src/components/form-fields/PreselectedRadioField.js [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
       {label}
       {info}
     </label>
 
     {error ? (
-      <div class="invalid-feedback">
-        <i class="fas fa-exclamation-triangle" />
-        <div class="field__msg  field__msg--error">{error}</div>{" "}
+      <div className={styles["invalid-feedback"]}>
+        <div className={classnames(styles["field__msg"], styles["field__msg--error"])}>{error}</div>{" "}
```
Comment (sc979) [inline www/front_src/src/components/form-fields/PreselectedRadioField.js]: same comment as checkboxstyle.js +46

File: www/front_src/src/components/form-fields/CheckboxField.js (hunk 1 above)
Comment (kduret) [reply www/front_src/src/components/form-fields/CheckboxField.js]: exclamation triangle did not work cause we didn't embed fontawesome

**Pull request merged**